core/io/networked_multiplayer_custom.cpp

```diff
@@ -139,9 +139,10 @@ NetworkedMultiplayerPeer::ConnectionStatus NetworkedMultiplayerCustom::get_conne
 //
 
 void NetworkedMultiplayerCustom::initialize(int p_self_id) {
-	if (connection_status != ConnectionStatus::CONNECTION_CONNECTING) {
-		return;
-	}
+	ERR_FAIL_COND_MSG(connection_status != ConnectionStatus::CONNECTION_CONNECTING,
+			"Can only initialize if connection status is CONNECTION_CONNECTING");
+	ERR_FAIL_COND_MSG(p_self_id < 0 || p_self_id > ~(1 << 31),
+			"Cannot initialize with invalid unique network id");
```

Comment:
Error message should end with a dot (might apply to other messages in this class).
```suggestion
			"Can only initialize if connection status is CONNECTION_CONNECTING.");
	ERR_FAIL_COND_MSG(p_self_id < 0 || p_self_id > ~(1 << 31),
			"Cannot initialize with invalid unique network id.");
```

Reply:
Thanks! I've updated those error messages, and the two that were added in PR https://github.com/godotengine/godot/pull/63628, so that they end with dots.